signer/src/bitcoin/packaging.rs

```diff
@@ -759,6 +774,29 @@ mod tests {
         assert!(!bag.can_fit_withdrawal_ids(&large_set));
     }
 
+    #[test]
+    fn item_order_matters_in_compute_optimal_packages() {
+        let mut rng = get_rng();
+        let len = rng.gen_range(10..=100) as usize;
```

Comment:
Although 10 is okay, I'd bump the minimum here to 25, to make it extraordinarily unlikely that we can have this test fail by chance.

Reply:
[42e7003](https://github.com/stacks-sbtc/sbtc/pull/1664/commits/42e70034d9a2d0f8f8ab861960cef2981a9b9bf4)